analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -56,7 +56,7 @@ import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.model.createAndLogIssue
 import org.ossreviewtoolkit.model.jsonMapper
 import org.ossreviewtoolkit.model.readValue
-import org.ossreviewtoolkit.spdx.SpdxLicense
+import org.ossreviewtoolkit.spdx.SpdxConstants
```

Comment:
I'd appreciate a generalization of the commit message like this:

```
Use NONE / NOASSERTION from SpdxConstants instead of SpdxLicense

These constants are not only used for licenses, so use them from
SpdxConstants as a preparation for eliminating them from SpdxLicense
later.
```

Reply:
Also, I believe this misses to replace `NOASSERTION` in `scanner/src/main/kotlin/scanners/ScanCode.kt`.

Reply:
Done.